#!/bin/sh /usr/share/dpatch/dpatch-run
## 101-magic-remove-awk.dpatch by Michael Piefel <piefel@debian.org>
##
## DP: Remove AWK detection by regular expression testing just for 'BEGIN:';
## DP: this caused a lot of false guesses (and problems with naughty programs
## DP: reading other programs' files), this closes: #308305, #375403 and makes
## DP: others disappear (for the time being, for example, it closes: #375722).

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/commands file-4.20/magic/Magdir/commands
--- file-4.20.orig/magic/Magdir/commands	2007-01-19 19:28:01.000000000 +0000
+++ file-4.20/magic/Magdir/commands	2007-03-27 12:23:14.000000000 +0000
@@ -28,7 +28,7 @@
 0	string/b	#!\ /bin/awk		awk script text executable
 0	string/b	#!\ /usr/bin/awk	awk script text executable
 # update to distinguish from *.vcf files
-0	regex		BEGIN[[:space:]]*[{]	awk script text
+#0	regex		BEGIN[[:space:]]*[{]	awk script text
 
 # AT&T Bell Labs' Plan 9 shell
 0	string/b	#!\ /bin/rc	Plan 9 rc shell script text executable
